src/pages/galleries/index.vue

```diff
@@ -11,15 +11,33 @@
           deck
           data-qa="gallery foyer"
         >
```

Comment:
Add checks on $fetchState.pending and $fetchState.error, to render the loading spinner and alert message component respectively.

Reply:
yes, good thinking. 
The loading spinner especially makes paginating much more user friendly.